tracer/src/Datadog.Trace/Span.cs

```diff
@@ -545,5 +548,19 @@ private void WriteCloseDebugMessage()
                 "Span closed: [s_id: {SpanId}, p_id: {ParentId}, t_id: {TraceId}] for (Service: {ServiceName}, Resource: {ResourceName}, Operation: {OperationName}, Tags: [{Tags}])\nDetails:{ToString}",
                 new object[] { Context.RawSpanId, Context.ParentIdInternal, Context.RawTraceId, ServiceName, ResourceName, OperationName, Tags, ToString() });
         }
+
+        internal SpanLink AddSpanLink(Span spanLinkToAdd, List<KeyValuePair<string, string>> attributes = null)
```

Comment:
nitpicky (and I think the last comment):

Maybe adding an XML doc comment for this would be beneficial - specifically the `<return>` callout of it being potentially `null`:
```suggestion
        /// <summary>
        /// Adds a <see cref="SpanLink"/> to this <see cref="Span"/>.
        /// </summary>
        /// <param name="spanLinkToAdd">The linked <see cref="Span"/>.</param>
        /// <param name="attributes">The attibutes for the created <see cref="SpanLink"/>.</param>
        /// <returns>The created <see cref="SpanLink"/>; otherwise, <see langword="null"/> when <see cref="IsFinished"/> is <see langword="true"/>.</returns>
        internal SpanLink AddSpanLink(Span spanLinkToAdd, List<KeyValuePair<string, string>> attributes = null)
```